device_posture_rule.go

```diff
@@ -165,41 +165,50 @@ type DevicePostureRuleMatch struct {
-	Infected         bool     `json:"infected,omitempty"`
-	IsActive         bool     `json:"is_active,omitempty"`
-	EidLastSeen      string   `json:"eid_last_seen,omitempty"`
-	RiskLevel        string   `json:"risk_level,omitempty"`
-	State            string   `json:"state,omitempty"`
-	LastSeen         string   `json:"last_seen,omitempty"`
+	ID               string               `json:"id,omitempty"`
+	Path             string               `json:"path,omitempty"`
+	Exists           bool                 `json:"exists,omitempty"`
+	Thumbprint       string               `json:"thumbprint,omitempty"`
+	Sha256           string               `json:"sha256,omitempty"`
+	Running          bool                 `json:"running,omitempty"`
+	RequireAll       bool                 `json:"requireAll,omitempty"`
+	CheckDisks       []string             `json:"checkDisks,omitempty"`
+	Enabled          bool                 `json:"enabled,omitempty"`
+	Version          string               `json:"version,omitempty"`
+	VersionOperator  string               `json:"versionOperator,omitempty"`
+	Overall          string               `json:"overall,omitempty"`
+	SensorConfig     string               `json:"sensor_config,omitempty"`
+	Os               string               `json:"os,omitempty"`
+	OsDistroName     string               `json:"os_distro_name,omitempty"`
+	OsDistroRevision string               `json:"os_distro_revision,omitempty"`
+	OSVersionExtra   string               `json:"os_version_extra,omitempty"`
+	Operator         string               `json:"operator,omitempty"`
+	Domain           string               `json:"domain,omitempty"`
+	ComplianceStatus string               `json:"compliance_status,omitempty"`
+	ConnectionID     string               `json:"connection_id,omitempty"`
+	IssueCount       string               `json:"issue_count,omitempty"`
+	CountOperator    string               `json:"countOperator,omitempty"`
+	TotalScore       int                  `json:"total_score,omitempty"`
+	ScoreOperator    string               `json:"scoreOperator,omitempty"`
+	CertificateID    string               `json:"certificate_id,omitempty"`
+	CommonName       string               `json:"cn,omitempty"`
+	ActiveThreats    int                  `json:"active_threats,omitempty"`
+	NetworkStatus    string               `json:"network_status,omitempty"`
+	Infected         bool                 `json:"infected,omitempty"`
+	IsActive         bool                 `json:"is_active,omitempty"`
+	EidLastSeen      string               `json:"eid_last_seen,omitempty"`
+	RiskLevel        string               `json:"risk_level,omitempty"`
+	State            string               `json:"state,omitempty"`
+	LastSeen         string               `json:"last_seen,omitempty"`
+	ExtendedKeyUsage []string             `json:"extended_key_usage,omitempty"`
+	CheckPrivateKey  bool                 `json:"check_private_key,omitempty"`
```

Comment:
you'll want this to be a `*bool` - see https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans